corehq/apps/userreports/v1/views.py

```diff
@@ -420,7 +421,7 @@ def page_context(self):
                     'is_static': self.existing_report.is_static,
                 })
             return self._handle_exception(error_response, e)
-        field_names = report_form.fields.keys()
+        field_names = list(report_form.fields.keys())
```

Comment:
would be better to keep this as a dict (hash lookup should be better than sequential scan of a list)

```py
field_names = report_form.fields
```